### ISSUE TITLE
As as User I'd like to have a Pause button. 

### ISSUE THREAD

**Issue by jackSzm** [novoda/peepz#24]
Problem : 
When I'm eating I don't want Peepz to make me pictures but I don't want to close the tab in the browser neither.

Solution : 
There should be a pause/resume button to stop picture taking functionality.

**Comment by tasomaniac** [comment]
This is related to #22 right?

**Comment by danybony** [comment]
💯 I'm using The Great Suspender to achieve that at the moment, but it would be GREAT to have a quick pause button.
To me this has more priority than #22 (even if it's related)

**Comment by ataulm** [comment]
Not sure it's related to #22 - pause/resume is different than the intervals the auto happens at.

One thing we discussed and wanted to avoid was to only have image updates for other people while you're sending out image updates. In the case of pause - when and why would it be used? Would you forget to resume it? Maybe a snooze for 15/20/30/60 minutes would cover the usecase? Or would there be an alert to remind you after a while?

workshop @ouchadam ✨

**Comment by tasomaniac** [comment]
My thinking was like pause ~ interval = Integer.MAX_INT

**Comment by jackSzm** [comment]
Yup I was thinking about snooze. And freezing other peepz during that time is 💯  to me.

**Comment by ataulm** [comment]
This is on android now 🎉  as "Off" when you select your picture take interval.
